app/controllers/users_controller.rb

```diff
@@ -2,7 +2,6 @@ class UsersController < ApplicationController
   before_action :require_no_user, only: [:new]
   before_action :require_user, only: %i(edit update save_settings)
   before_action :set_user, only: %i(info followed following followers)
```

Comment:
```suggestion
  before_action :set_user, only: %i(info followed following followers)
  
```

Reply:
Please commit this one too. Thanks!

Reply:
@gautamig54 after this one, feel free to mention me on Gitter as this PR is ready.